pkg/neorpc/rpcevent/filter_test.go

```diff
@@ -263,7 +263,7 @@ func TestMatches(t *testing.T) {
 			name: "notary request, signer mismatch",
 			comparator: testComparator{
 				id:     neorpc.NotaryRequestEventID,
-				filter: neorpc.TxFilter{Signer: &badUint160},
+				filter: neorpc.NotaryRequestFilter{Signer: &badUint160},
```

Comment:
Please, pay attention to the test structure. You need to add one more testcase and adjust `notary request, filter match` testcase.